Source/Blazorise/Components/ColorEdit/ColorEdit.razor.cs

```diff
@@ -22,6 +23,14 @@ public partial class ColorEdit : BaseInputComponent<string>, ISelectableComponen
     /// <inheritdoc/>
     public override async Task SetParametersAsync( ParameterView parameters )
     {
+        if ( Rendered )
+        {
+            if ( parameters.TryGetValue<string>( nameof( Color ), out var paramColor ) && !paramColor.IsEqual( Color ) )
+            {
+                ExecuteAfterRender( Revalidate );
```

Comment:
I was thinking, isn't this contradictory? If Rendered, ExecuteAfterRender? Wasn't the whole point of it to make sure logic only executes after the component has been rendered, for example when js interoping? Does this still make sense like this, here? Meaning, you would only need to call `Revalidate` here?

Reply:
Revalidate would try to validate immediately, and there is no guarantee that the component is already fully initialized.

Reply:
then PR LGTM. I'm hoping we aren't forgetting any component, right? :D

Reply:
> I'm hoping we aren't forgetting any component, right? :D

We'll know it some day :D